src/mlpack/tests/xgboost_test.cpp

```diff
@@ -1,6 +1,6 @@
 /**
  * @file tests/xgboost_test.cpp
- * @author Rishabh Garg
+ * @author Abhimanyu Dayal
```

Comment:
Since this is shared code, I would add you as an additional author, what do you think?

Reply:
Yeah sure, this works!

Reply:
Actually, his original work in this file was a redundant to my implementation, so there isn't any shared code in this file. 
He had originally written an SSE loss function, and test cases for that, which I'm not using in my XGBoost implementation.